TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst

```diff
@@ -751,6 +751,7 @@ overriding the target attribute for external links and resource links.
 :externalLinkTarget: (string) Override the target attribute for external links, defaults to ``_blank``. Can be disabled with an empty value.
 :resourceLinkTarget: (string) Override the target attribute for resource links, defaults to ``_blank``. Can be disabled with an empty value.
 :forceConversion: (boolean) Whether to convert URIs in a non-live workspace, defaults to ``FALSE``
+:absolute: (boolean) can be used to convert node uris to absolute links, defaults to ``FALSE``
```

Comment:
tiny nitpick, would be better if the sentence was started with uppercase, like the other do